### PR TITLE
`upgrade` command

### PR DESCRIPTION
**Pull request by MaksymilianDemitraszek**
Maybe in the future: https://github.com/software-mansion/protostar/issues/54

### CODE REVIEW

File: src/commands/upgrade/upgrade.py [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+        self.current_version = self.get_current_version()
+        self.latest_version_tag = self.get_latest_release()["tag_name"]
+        self.latest_version = version.parse(self.latest_version_tag)
+
+    def is_current_latest(self):
+        return self.latest_version == self.current_version
+
+    def upgrade(self):
+        logger.info("Looking for a new version ...")
+        if self.latest_version <= self.current_version:
```
Comment (kasperski95) [inline src/commands/upgrade/upgrade.py:47]: I am not sure whether this condition will always work as expected e.g. "0.0.10" < "0.0.9"

File: src/commands/upgrade/upgrade.py [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+        raise UpgradeManagerException(f"{platform} is not supported")
+
+    @classmethod
+    def get_latest_release(cls):
+        headers = {"Accept": "application/json"}
+        response = requests.get(f"{PROTOSTAR_REPO}/releases/latest", headers=headers)
+        return response.json()
+
+    def get_current_version(self):
+        path = self.protostar_dir / "dist" / "protostar" / "info" / "pyproject.toml"
```
Comment (kasperski95) [inline src/commands/upgrade/upgrade.py:113]: Nice. I always used `os.path.join`.

**Review by kasperski95 — CHANGES_REQUESTED** [review]
Requesting changes due to:
- comparing tags as strings

File: src/commands/upgrade/upgrade.py (hunk 1 above)
Comment (MaksymilianDemitraszek) [reply src/commands/upgrade/upgrade.py:47]: We do not compare them as strings but as parsed version objects

Comment (kasperski95) [reply src/commands/upgrade/upgrade.py:47]: Yep, my bad. I checked the type of `latest_version_tag` rather than `latest_version`.

**Review by kasperski95 — APPROVED** [review]

**Pull request merged**